src/main/java/com/exadel/discountwebapp/discount/entity/Discount.java

```diff
@@ -64,6 +64,9 @@
     @Column(name = "dis_promocode", length = 100)
     private String promocode;
 
+    @Column(name = "dis_viewed")
+    private Long quantityViews;
```

Comment:
what does this field mean?

Reply:
this is not a valid English

it should be viewQuantity

But better to use viewCount or viewNumber

Reply:
fixed